dashboard/src/components/ui/sheet.tsx

```diff
@@ -38,7 +38,7 @@ const sheetVariants = cva(
           'inset-x-0 bottom-0 border-t data-[state=closed]:slide-out-to-bottom data-[state=open]:slide-in-from-bottom',
         left: 'inset-y-0 left-0 h-full w-3/4 border-r data-[state=closed]:slide-out-to-left data-[state=open]:slide-in-from-left sm:max-w-sm',
         right:
-          'inset-y-0 right-0 h-full w-3/4  border-l data-[state=closed]:slide-out-to-right data-[state=open]:slide-in-from-right sm:max-w-sm',
+          'inset-y-0 right-0 h-full w-3/4  border-l data-[state=closed]:slide-out-to-right data-[state=open]:slide-in-from-right-[50%] sm:max-w-sm',
```

Comment:
The log viewer is still a big "laggy" to me, like it kind of spawns in the edge of the screen and then starts sliding. It's worse when closing. For the last part you should probably alter `data-[state=closed]:slide-out-to-right` as well

Reply:
```suggestion
          'inset-y-0 right-0 h-full w-3/4  border-l data-[state=closed]:slide-out-to-right-[100%] data-[state=open]:slide-in-from-right-[100%] sm:max-w-sm',
```
Seems to have fixed it